components/dashboards-web-component/src/utils/apis/DashboardAPI.jsx

```diff
@@ -144,4 +144,16 @@ export default class DashboardAPI {
             .getHTTPClient()
             .get(`/report-config`);
     }
+
+    /**
+     * Export Dashboard
+     *
+     * @param {string} dashboardId Dashboard ID
+     * @returns {Promise} Promise
+     */
+    static ExportDashboardByID(dashboardId) {
```

Comment:
Method/fuction name should have the camelCase. See: https://www.w3schools.com/js/js_conventions.asp

Reply:
Fixed in #5aa031f